README.md

```diff
@@ -1,6 +1,8 @@
-[![license: CC BY-SA 3.0](https://licensebuttons.net/l/by-sa/3.0/80x15.png)](https://creativecommons.org/licenses/by-sa/3.0/)
+[![DOI](https://zenodo.org/badge/566711963.svg)](https://zenodo.org/badge/latestdoi/566711963)
 [![downloads](https://img.shields.io/github/downloads/ITISFoundation/IXI025/total.svg)](https://github.com/ITISFoundation/IXI025/releases)
 
+[![license: CC BY-SA 3.0](https://licensebuttons.net/l/by-sa/3.0/80x15.png)](https://creativecommons.org/licenses/by-sa/3.0/)
```

Comment:
are you sure the licence button is correct?
 why dont' you use https://shields.io/??
<img width="1011" alt="image" src="https://user-images.githubusercontent.com/32402063/202473423-9c545157-ebb3-4327-a563-abf82a74bce1.png">

Reply:
![image](https://user-images.githubusercontent.com/12702862/202474780-338291a1-dfba-45b7-9624-9bb4f538b0b9.png)

Reply:
Its what the badge looks like (ugly I know): https://gist.github.com/lukas-h/2a5d00690736b4c3a7ba